arangod/RocksDBEngine/RocksDBCollection.cpp

```diff
@@ -837,20 +840,22 @@ Result RocksDBCollection::read(transaction::Methods* trx,
       break;
     }  // else found
 
-    std::string* buffer = result.setManaged();
-    rocksdb::PinnableSlice ps(buffer);
     res = lookupDocumentVPack(trx, documentId, ps, /*readCache*/true, /*fillCache*/true);
     if (res.ok()) {
-      if (ps.IsPinned()) {
-        buffer->assign(ps.data(), ps.size());
-      } // else value is already assigned
-      result.setRevisionId(); // extracts id from buffer
+      cb(documentId, VPackSlice(reinterpret_cast<uint8_t const*>(ps.data())));
     }
   } while (res.is(TRI_ERROR_ARANGO_DOCUMENT_NOT_FOUND) &&
            RocksDBTransactionState::toState(trx)->setSnapshotOnReadOnly());
   return res;
 }
 
+// read using a token!
```

Comment:
is this a TODO?

Reply:
aha! It is just an old but copied/moved comment. 
We either can remove it or adjust it to actually fit. Because no token is used here, but instead it is `LocalDocumentId` now.